examples/basic-routing/index.tsx

```diff
@@ -42,4 +42,9 @@ const App = () => {
   );
 };
 
-render(<App />, document.getElementById('root'));
+const container = document.getElementById('root');
+if (!container)
+  throw new Error('No root element found to render basic routing example');
+
+const root = createRoot(container);
+root.render(<App />);
```

Comment:
How are we testing if it works with [StrictMode](https://react.dev/reference/react/StrictMode) ?

Reply:
See integration tests file

Reply:
yeahhh, I was looking at it now